src/test/java/com/scottlogic/AuthorPostSorterTest.java

```diff
@@ -0,0 +1,76 @@
+package com.scottlogic;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+
+import java.time.OffsetDateTime;
+import java.time.ZoneOffset;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class AuthorPostSorterTest {
+    @Test
+    void nullTest() {
```

Comment:
As per the talk with Matt a good style for test names is `function_inputValues_expectedOutput` as this makes it clearer when it flags as a failure when running the full test suite.

Reply:
Yeah makes sense